sdk/storage/storage-queue/README.md

```diff
@@ -93,10 +85,15 @@ For example, you can create following CORS settings for debugging. But please cu
 
 ## Examples
 
-### Import types
+### Import the package
 
-You can use the `const Azure = require("@azure/storage-queue");` shown above then use types and functions from `Azure`.
-Or you can selectively import certain types,
+To use the clients, import the package into your file:
+
+```javascript
+const AzureStorageQueue = require("@azure/storage-queue");
+```
+
+Alternatively, selectively import only the types you need:
 
 ```javascript
 const { QueueServiceClient, SharedKeyCredential } = require("@azure/storage-queue");
```

Comment:
`SharedKeyCredential` -> `StorageSharedKeyCredential`

Can you please do a favour by updating all the instances of this in the readme?